### PR TITLE
Makes spontaneous appendicitis rarer

### PR DESCRIPTION
**Pull request by Alberyk**
What it says in the title.

### CODE REVIEW

**Review by fernerr — CHANGES_REQUESTED** [review]
I feel something like appendicitis isn't severe enough to be listed under the major category, moderate is more appropriate. It'd still have less chance than "communications blackout" of appearing, which feels quite rare enough.

**Comment by JohnWildkins** [comment]
I'm not sure I even agree with it being in moderate, tbh, but definitely not in major. Bear in mind that an event being in a category means that it's fighting with the others to be picked every cycle, and I don't think one crewmember getting appendicitis should be on the same tier as a blob or meteor wave.

**Comment by Alberyk** [comment]
It is a moderate event now.

File: code/modules/events/event_container.dm [hunk 1]
```diff
@@ -166,7 +165,8 @@ var/global/list/severity_to_string = list(EVENT_LEVEL_MUNDANE = "Mundane", EVENT
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Random Antagonist",				/datum/event/random_antag,		 			0,		list(ASSIGNMENT_ANY = 1, ASSIGNMENT_SECURITY = 1),0,10,125, list("Extended")),
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Rogue Drones",						/datum/event/rogue_drone, 					50,		list(ASSIGNMENT_SECURITY = 25)),
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Moderate Spider Infestation",		/datum/event/spider_infestation/moderate,	50,		list(ASSIGNMENT_SECURITY = 25)),
-		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Moderate Vermin Infestation",		/datum/event/infestation/moderate, 			30,		list(ASSIGNMENT_JANITOR = 15, ASSIGNMENT_SECURITY = 15, ASSIGNMENT_MEDICAL = 15))
+		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Moderate Vermin Infestation",		/datum/event/infestation/moderate, 			30,		list(ASSIGNMENT_JANITOR = 15, ASSIGNMENT_SECURITY = 15, ASSIGNMENT_MEDICAL = 15)),
+		new /datum/event_meta(EVENT_LEVEL_MAJOR, "Appendicitis", 						/datum/event/spontaneous_appendicitis, 		0,		list(ASSIGNMENT_MEDICAL = 5))
```
Comment (fernerr) [inline code/modules/events/event_container.dm:169]:
```suggestion
		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Appendicitis", 						/datum/event/spontaneous_appendicitis, 		0,		list(ASSIGNMENT_MEDICAL = 5))
```

**Review by fernerr — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**